### PR TITLE
Feature/trending gui

### PR DESCRIPTION
**Pull request by ichorid**
This PR adds the "Popular" tab, showing top torrents. The icon is courtesy of @drew2a 
llk

### CODE REVIEW

**Comment by drew2a** [comment]
@ichorid you are always welcome.

I'm sure that 🔥 is not a brilliant match for "popular", but it is the best I could find.

If anyone could suggest something better, that would be nice.

**Comment by devos50** [comment]
Finding a matching icon to a menu item is always hard. I like the flame icon! It's also what Reddit uses to sort within a subreddit (hottest posts) 👍

**Review by drew2a — DISMISSED** [review]
I've checked the PR locally and it works just fine 👍

File: src/tribler-core/tribler_core/modules/metadata_store/restapi/channels_endpoint.py [hunk 1]
```diff
@@ -43,9 +43,17 @@ def setup_routes(self):
                 web.put(r'/{channel_pk:\w*}/{channel_id:\w*}/torrents', self.add_torrent_to_channel),
                 web.post(r'/{channel_pk:\w*}/{channel_id:\w*}/commit', self.post_commit),
                 web.get(r'/{channel_pk:\w*}/{channel_id:\w*}/commit', self.is_channel_dirty),
+                web.get('/popular_torrents', self.get_popular_torrents_channel),
             ]
         )
 
+    def add_download_progress_to_metadata_list(self, contents_list):
```
[the remark this answers is not in the document]
Comment (ichorid) [reply src/tribler-core/tribler_core/modules/metadata_store/restapi/channels_endpoint.py:50]: This one is related to every endpoint that returns torrent metadata. It adds local downloads info to the response.

**Review by drew2a — DISMISSED** [review]

**Review by drew2a — APPROVED** [review]

**Pull request merged**